libs/solid/quiz/src/lib/components/start/start.component.html

```diff
@@ -17,16 +15,54 @@
       Fragenanzahl kann hier festgelegt werden:
     </p>
   </mat-card-content>
-  <mat-card-actions *ngIf="QuizLoaded | async">
-    <mat-form-field>
-      <input
-        [formControl]="questionCount"
-        matInput
-        placeholder="Anzahl an Fragen"
-        type="number"
-      />
+  <hr />
+  <mat-card-title>Konfiguration</mat-card-title>
+  <mat-card-actions *ngIf="metaData$ | async as meta">
+    <mat-form-field appearance="outline">
+      <mat-label>Tags</mat-label>
+      <mat-chip-list #tagList>
+        <mat-chip
+          *ngFor="let tag of chosenTags"
+          (removed)="removeTag(tag)"
+          selected
+        >
+          {{ tag }}
+          <button matChipRemove>
+            <mat-icon>cancel</mat-icon>
+          </button>
+        </mat-chip>
+        <input
+          type="text"
+          placeholder="More Tags..."
+          [matChipInputFor]="tagList"
+          (matChipInputTokenEnd)="addTag($event)"
+          [matAutocomplete]="auto"
+        />
+      </mat-chip-list>
+      <mat-autocomplete
+        #auto="matAutocomplete"
+        (optionSelected)="selected($event)"
+      >
+        <mat-option *ngFor="let tag of meta.tags" [value]="tag">
+          {{ tag }}
+        </mat-option>
+      </mat-autocomplete>
+    </mat-form-field>
+
+    <mat-form-field appearance="outline">
+      <mat-label>Anzahl der Fragen</mat-label>
+      <input [formControl]="questionCount" matInput type="number" />
     </mat-form-field>
+
+    <mat-slider min="0" max="5" step="1" [(ngModel)]="difficulty"> </mat-slider>
```

Comment:
Hier fehlt noch ein Label für den Slider

Reply:
also - eine Überschrift und dann die Information, was gewählt wurde. Mir gefällt das _thumb label_ vom Design her nicht so gut, aber ich denke, es ist vermutlich der beste Weg.

Hmm... ist es nicht sinnvoll, die Anzahl der Fragen auch als Slider zu designen? Es ist auch nur eine numerische Auswahl. Ich glaube, das würde besser aussehen.

Reply:
Oder wir können ein kleines Feld neben dem Slider erstellen, in dem die "Level" (easy, normal, hard,...) angezeigt ist. Was denkst du ?

Ich finde den Slider auch schöner und eleganter

Reply:
Slider mit Anzeige des Levels finde ich gut, aber wir haben uns jetzt für eine numerische Angabe entschieden: Manche Content-Teams werden nur 3 Levels nutzen, andere vielleicht 5. Deswegen wird die Übersetzung in "einfach, mittel, schwer" nicht überall gleich. Da ist dann eine rein numerische Angabe sicherer. Aber ich würde das mal im Kopf behalten bzw. Kerstin mal um eine Meinung fragen.

Und Slider für die Anzahl sollten wir dann machen!